todo/src/App.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
A good way to start working through your redux is creating components in this order

index => store => reducer => action => component